### PR TITLE
Gra 1088 dns updates

### PR DESCRIPTION
**Pull request by mattkasun**

### CODE REVIEW

File: mq/publishers.go [hunk 1]
```diff
@@ -160,6 +160,226 @@ func ServerStartNotify() error {
...
+		if node.Address.IP != nil {
+			dns.Address = node.Address.IP.String()
+			alldns = append(alldns, dns)
+		}
+		if node.Address6.IP != nil {
+			dns.Address = node.Address6.IP.String()
+			alldns = append(alldns, dns)
+		}
+	}
+	clients, err := logic.GetNetworkExtClients(newnode.Network)
```
Comment (0xdcarns) [inline mq/publishers.go:217]: from 217-250, could probably be another function to getAllDNS or something

File: mq/publishers.go [hunk 2]
```diff
@@ -160,6 +160,226 @@ func ServerStartNotify() error {
...
+		Name:    old.ClientID + "." + network,
+		NewName: new.ClientID + "." + network,
+	}
+	if err := PublishDNSUpdate(network, dns); err != nil {
+		return err
+	}
+	return nil
+}
+
+// PublishDeleteExtClient publish dns update to delete extclient entry
```
Comment (0xdcarns) [inline mq/publishers.go:338]: incorrect Comment

File: mq/publishers.go [hunk 3]
```diff
@@ -160,6 +160,226 @@ func ServerStartNotify() error {
...
+			errMsgs.ErrorStrings = append(errMsgs.ErrorStrings, err.Error())
+		}
+	}
+	if len(errMsgs.ErrorStrings) > 0 {
+		return errMsgs
+	}
+	return nil
+}
+
+// PublishExtClientUpdate publishes dns update for extclient name change
```
Comment (0xdcarns) [inline mq/publishers.go:325]: incorrect comment

**Review by 0xdcarns — CHANGES_REQUESTED** [review]

**Comment by theguy951357** [comment]
advised to wait.

**Review by 0xdcarns — APPROVED** [review]

**Review by afeiszli — APPROVED** [review]

**Review by theguy951357 — APPROVED** [review]
followed instructions on GRA-1088
all steps pass

**Pull request merged**